pandas/core/groupby/groupby.py

```diff
@@ -1399,25 +1399,10 @@ def _op_via_apply(self, name: str, *args, **kwargs):
         f = getattr(type(self._obj_with_exclusions), name)
         sig = inspect.signature(f)
 
-        if "axis" in kwargs and kwargs["axis"] is not lib.no_default:
-            axis = self.obj._get_axis_number(kwargs["axis"])
-            self._deprecate_axis(axis, name)
-        elif "axis" in kwargs:
-            # exclude skew here because that was already defaulting to lib.no_default
-            #  before this deprecation was instituted
-            if name == "skew":
-                pass
-            elif name == "fillna":
-                # maintain the behavior from before the deprecation
-                kwargs["axis"] = None
-            else:
-                kwargs["axis"] = 0
-
         # a little trickery for aggregation functions that need an axis
         # argument
         if "axis" in sig.parameters:
-            if kwargs.get("axis", None) is None or kwargs.get("axis") is lib.no_default:
-                kwargs["axis"] = self.axis
+            kwargs["axis"] = self.axis
```

Comment:
Will this be able to be removed once we remove `axis` from `groupby`?

Reply:
Yep! That's coming next.